ci-operator/step-registry/openshift-qe/installer/featuregate/openshift-qe-installer-featuregate-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat << EOF| oc replace -f -
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: CustomNoUpgrade 
+  customNoUpgrade:
+    enabled:
+    - ${FEATURE}
```

Comment:
@jtaleric mentioned he did this because on a 24 nodes cluster enabling ALL feature gates takes like 1.5hrs for cluster to stabilize, since each operator probably is enabling these 1 by 1 ? this is why he had to do this 1 feature gate enablement - the one we care about instead of creating a TechPreviewNoUpgrade cluster..

Did I get this right? This is the reason why we don't use the techpreview job workflows right?

Reply:
We wanted to do the opposite.. 4.18-dev-preview (or any master equivalent) enabled restricted PodSecurity Standard.. wanted to disable that to test for 4.18 LTS GA equivalent that my team will need to support.

So I guess I will have to add a step here for disabled feature.. hopefully that will work.

Reply:
https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1729699499772989

Reply:
@jtaleric what's up dude! thanks for the help 0/